testapp/src/main/java/com/rakuten/tech/mobile/testapp/helper/MiniAppListHelper.kt

```diff
@@ -0,0 +1,43 @@
+package com.rakuten.tech.mobile.testapp.helper
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+import com.rakuten.tech.mobile.miniapp.MiniAppInfo
+import java.lang.IllegalStateException
+
+class MiniAppListHelper(context: Context) {
```

Comment:
I think this can be refactored to the specific purpose of this class. What do you think?
A similar concept applies to `Util`. https://kotlinlang.org/docs/reference/coding-conventions.html#source-file-names

Reply:
How about `MiniAppListStorage`?

Reply:
`MiniAppListStore` sounds okay to me.